tools/cli/src/opt/kanidm.rs

```diff
@@ -594,8 +594,7 @@ pub enum ServiceAccountPosix {
 pub struct PersonUpdateOpt {
     #[clap(flatten)]
     aopts: AccountCommonOpt,
-    #[clap(long, short, help = "Set the legal name for the person.",
-    value_parser = clap::builder::NonEmptyStringValueParser::new())]
+    #[clap(long, short, help = "Set the legal name for the person. Use '' to clear.")]
```

Comment:
```suggestion
    #[clap(long, short, help = "Set the legal name for the person. Pass empty text to clear.")]
```

```suggestion
    #[clap(long, short, help = "Set the legal name for the person, or remove it via empty string.")]
```

```suggestion
    #[clap(long, short, help = "Set, or remove (via empty value), the legal name for the person.")]
```

```suggestion
    #[clap(long, short, help = "Set the legal name for the person. An empty value clears the field.")]
```

Might be a bit clearer to what you're actually supposed to do here as opposed to the somewhat obtuse `''`. I can't come up with a wording I'm 100% happy with that is still concise so here are 4 variants. The last I like best but it's also the longest.